content/en/blog/sysflow-falco-sidekick.md

```diff
@@ -0,0 +1,146 @@
+
+For example, SysFlow embeds Falco's rich observability libraries into its cloud-native security telemetry stack and Falco's rules language to achieve data abstraction, behavioral analytics, and noise reduction. 
+
+The Falco project and the community helped make the SysFlow project a reality and we're excited to guide you through a deep dive today! Also, don't miss how we integrated our solution with Falcosidekick to help diagnose alerts.
+
+### What is SysFlow?
+
+SysFlow is a compact open telemetry [format](https://sysflow.readthedocs.io/en/latest/spec.html) that records workload behaviors by connecting event and flow representations of process control flows, file interactions, and network communications. The resulting abstraction encodes a graph structure that enables provenance reasoning on host and container environments, and fast retrieval of security-relevant information.
+
+<br/>
+
+![image description](/img/sysflow-falco-sidekick/01-diagram-outputs.png)
+
+<br/>
+
+### What challenges does it solve?
+
+These are the keys that sets SysFlow apart:
+
+* It drastically **reduces data footprints** when compared to raw system call collection.
+* **Reduces event fatigue** (a.k.a. "too many alerts") thanks to its underlying entity-relational model and flow abstractions.
+* At the same time, it **minimizes the requirements for hard filters**.
+* It **provides useful context** by linking together system event data at the data format level. This speeds up security analysis, enabling analytics beyond simple manual policies, including machine learning, stateful analytics, and automated policy generation.
+* It treats system security monitoring as a data science problem through **rapid edge analytics building**, easy-to-use APIs, and support for multiple data serialization and export formats.
+
+
+### Built on Falco libs
+
+The SysFlow format is supported by an [edge processing pipeline](https://github.com/sysflow-telemetry/sf-processor) that aggregates event provenance information and supports real-time enrichment of the telemetry stream with attack TTP labels, environment metadata, and Kubernetes log data.
+
+As a benefit of using the Sysflow format, users go from managing individual events generated from different sources to obtaining an enhanced dataset to work with, enabling them to focus on writing and sharing analytics on a scalable, common open-source platform.
+
+The framework builds on [Falco libs](https://github.com/falcosecurity/libs) and the [Falco rules language](https://falco.org/docs/rules/) to create the plumbing required for system telemetry as shown in the diagram below.
+
+Falco libs facilitate the collection of system events generated from the host and containers through the eBPF probe in the kernel space.
+
+At user level, The SysFlow Collector, also leveraging Falco libs, collects the previous events and exports the captured information using Apache Avro object serialization
```

Comment:
At user level, The SysFlow Collector, also leveraging Falco libs, collects the previous events and exports the captured information using Apache Avro object serialization.